### PR TITLE
Add GitHub Enterprise Agreement

### PR DESCRIPTION
**Pull request by vollmera**
Adding the GitHub Enterprise Agreement to the Site Policy repository.

### CODE REVIEW

**Comment by vollmera** [comment]
Closing this PR because it's a new policy so there is no diff showing changes to comment on. Along with the other updates, this pull request will be reopened and merged on April 19.

**Comment by meentastic** [comment]
We've decided to reopen this PR as we are including changes to the [posted policy](https://help.github.com/en/articles/github-enterprise-agreement).  This update to the Enterprise Agreement includes a number of changes for compliance purposes and clarifications, including changes to the following sections:

- Definitions
- Account Terms
- Acceptable Use
- Private Repositories
- Additional Terms for GitHub Pages and Learning Lab
- Cancellation and Termination
- Limitation of Liability
- Defense of Claims; Release
- Confidentiality
- Miscellaneous

File: Policies/github-enterprise-agreement.md [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+
+**3.2 Content Restrictions.** Under no circumstances will you or your Users upload, post, host, or transmit any Content to your repositories that:
+
+-   is unlawful or promotes unlawful activities;
+
+-   is or contains sexually obscene content;
+
+-   is libelous, defamatory, or fraudulent;
+
+-   •	gratuitously depicts or glorifies violence, including violent images;
```
Comment (meentastic) [inline Policies/github-enterprise-agreement.md]: note to remove extra bullet.

**Comment by bluemazzoo** [comment]
Thank you all for your comments and feedback! As stated in our [blog post](https://github.blog/2019-03-14-githubs-site-policy-updates-are-ready-for-your-feedback/), the comment period ends today. I am closing this pull request for internal review of any remaining comments. We will commit final clarifications or edits when we merge this pull request next Friday, April 19. At that time, the updated version of this policy will be effective.

**Pull request closed without merging**